src/include/ceph_fs.h

```diff
@@ -430,6 +430,12 @@ enum {
 	CEPH_MDS_OP_RDLOCK_FRAGSSTATS = 0x01507
 };
 
+#define IS_CEPH_MDS_OP_NEWINODE(op) (op == CEPH_MDS_OP_CREATE     || \
+				     op == CEPH_MDS_OP_MKNOD      || \
+				     op == CEPH_MDS_OP_MKSNAP     || \
```

Comment:
For `CEPH_MDS_OP_MKSNAP` is special and won't create new inode in `MDS`. We can just skip it.

Reply:
yes, thanks!